ci/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+
+cargo build --release --target $TARGET
+cargo build --release -p multirust-dist --target $TARGET
+
+if [ -z "$SKIP_TESTS" ]; then
+  cargo test --release --target $TARGET
+fi
```

Comment:
Oh oops I mis-transcribed the original script!

Also good point about the dist scripts, I'll double check to make sure it all checks out